Tasks/Common/SonarQubeHelpers/SonarQubeHelper.ps1

```diff
@@ -221,7 +221,7 @@ function IsPrBuild
 
 function GetSonarQubeBuildDirectory
 {
-    $agentBuildDirectory = GetTaskContextVariable "Agent.BuildDirectory" 
+    $agentBuildDirectory = $env:Agent_BuildDirectory 
```

Comment:
We should only access the task variables using the context. The agent variables are env variables.